src/lib/accessibility/Behaviors/Button/ToggleButtonBehavior.ts

```diff
@@ -0,0 +1,11 @@
+import { Accessibility } from '../../interfaces'
+
+export const ToggleButtonBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    root: {
+      role: 'button',
+      'aria-pressed': !!props['active'],
+      'aria-disabled': !!props['disabled'],
```

Comment:
As for me 'disabled' more refers to a state than props. I think behaviors might also need to accept state. So we can properly transform/ validate each aria-* attributes before rendering.
Also, we can consider creating a getter method for attributes, which will use props and state to validate them. @jurokapsiar what do you think?

Reply:
Oh, got it! thx for the explanation. It does make sense.